GitUI/UserControls/FilterToolBar.cs

```diff
@@ -166,18 +166,6 @@ public void InitToolStripStyles(Color toolForeColor, Color toolBackColor)
             tstxtRevisionFilter.ForeColor = toolForeColor;
         }
 
-        public void PreventToolStripSplitButtonClosing(ToolStripSplitButton? control)
-        {
-            if (control is null
-                || Parent is not ContainerControl parentContainer)
-            {
-                return;
-            }
-
-            control.Tag = parentContainer.FindFocusedControl();
```

Comment:
Should this be removed?
It changes focusing when menu has changed

Reply:
Yes, this was part of the dirty hack, in a previous version, giving focus to a focusabled control was preventing the menus from collapsing. Don't ask me why ^^, but it used to work. That's what was really not satisfying. Now it's gone, and it's good.